packages/farms/constants/arb.ts

```diff
@@ -1,6 +1,7 @@
 import { arbitrumTokens } from '@pancakeswap/tokens'
 import { FeeAmount } from '@pancakeswap/v3-sdk'
-import { FarmConfigV3 } from '../src'
+import { getAddress } from 'viem'
+import { FarmConfigV3, SerializedFarmConfig } from '../src'
```

Comment:
pls import directly from types

Reply:
import { FarmConfigV3, SerializedFarmConfig } from '../src/types'
like this ?